### DISCUSSION TITLE
Should we write about old features that any current browsers does (or doesn't) implement?

### DISCUSSION THREAD

**Discussion by NagayamaToshiaki** [mdn/mdn-community#589] [Content] [answered]
Since Japanese is my mother tongue, English translation is not accurate. So, I put Japanese claims after it.

For example, "useCapture (of the addEventListener method) has not always been optional", which is false after IE9. These kinds of information are just the source of the confusion to most of the new comers, which only maintain for current browsers.
I hope you update the articles to up to date (at least for IE11) or write those deprecated informations at the end of the article they were on as a sidenote (for historic interest).

例えばaddeventListenerメソッドの「useCapture はどんなブラウザーでもオプションになっているわけではありません。」という記述はIE9以降は当てはまりません。このような情報は現行ブラウザの未対応している新参者にとっては混乱のもとでしかありません。
現行の（少なくともIE11まで対応の）情報だけを載せるか、古い情報は（歴史的興味として）記事末尾に参考情報として載せたほうが良いのではないでしょうか。

**Comment by wbamberg** [comment]
I agree with you, we should remove this note and should not in general document features of obsolete browsers.

The mdn/browser-compat-data project agreed a policy where (I think) if a feature had not been supported by any browsers for >2 years, we should delete any docs for it. mdn/content has been tending to follow this policy as well.

I can never find where this policy is documented though.

**Reply by wbamberg** [reply, marked as answer]
-> https://github.com/mdn/content/pull/6865 removes the note.

**Comment by teoli2003** [comment]
browser-compat-data documents this policy [there](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#removal-of-irrelevant-features).

Reviewers of mdn/content applies the same, as far as I noticed when deleting content here. (But I don't think this is documented anywhere)

**Reply by Rumyra** [reply]
This I believe is what we have written about it

https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Conventions_definitions

But there's no mention of when to remove things. I'll add it to my todos to update with the above policy @wbamberg mentioned

**Reply by hamishwillee** [reply]
FWIW there have been cases where removing BCD made sense, but removing the MDN docs did not. In particular the rules can mean that there are things in the spec that are not in BCD (for good reasons) and this can be confusing for readers.

Would it make sense for BCD to maintain a public list of removals with reasons on MDN? that would allow searchable reason for removal in just one place.

This is a bit similar to the ongoing  discussion on "if we deprecate something can we indicate what it was deprecated in favour of". To me the answer is "of course we should", but there is no formal hint in macros or other way to find out what happened without a lot of work.

P.S. Edited, note I am strongly in favour of these rules, I just want an audit trail.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
